baseplate/lib/message_queue.py

```diff
@@ -72,7 +72,7 @@ def get(self, timeout: Optional[float] = None) -> bytes:
             duration of the call.
 
         """
-        for time_remaining in RetryPolicy.new(budget=timeout):
+        for _, time_remaining in RetryPolicy.new(budget=timeout):
```

Comment:
🔕 this is likely also a bug in the current code that the value is not always time_remaining, but I think it's out of the scope of this PR.

Reply:
what are you seeing instead of the time remaining or None?